src/components/sections/LatticeView.tsx

```diff
@@ -226,13 +233,9 @@ const LatticeView = ({ compilationResult, repeats = 1 }: LatticeViewProps): JSX.
                             setCellDimension={setCellDimensionPixels}
                         />
                     </Box>
-                    <Box
-                        w="100vw"
-                        h={slices_len * cellDimensionPixels * (repeats + 1)}
-                        overflow="hidden"
-                    >
+                    <Box h={slices_len * cellDimensionPixels * (repeats + 1)} overflow="hidden">
                         <Draggable ref={nodeRef}>
-                            <Box id="lattice-container">
+                            <Box id="lattice-container" w={largeLattice() ? "fit-content" : "94vw"}>
```

Comment:
Now this happens:
![Screenshot_20220303_004301](https://user-images.githubusercontent.com/36427091/156528928-a1d38efa-aaef-46ef-8ca9-d74eb1ac0cbc.png)

Reply:
Does leaving it always at `"94vw"` not work?

Reply:
Does that happen after you resize the window, or right from the start? It seems like you might be using a large monitor, or one with a high pixel count width? When the lattice is large, it should expand freely off the screen to the right rather than getting truncated at the screens edge, hence why it has two paradigms, one for small lattices and one for large

Reply:
And are you still zooming with ctrl+scroll?